XCUITests/SearchTest.swift

```diff
@@ -218,4 +218,38 @@ class SearchTests: BaseTestCase {
         waitForExistence(app.textFields["url"], timeout: 20)
         waitForValueContains(app.textFields["url"], value: "google")
     }
+    
+    func testSearchIconOnAboutHome() {
+        waitForTabsButton()
+        navigator.performAction(Action.OpenNewTabFromTabTray)
+        waitForTabsButton()
+        
+        // Search icon is displayed.
+        waitForExistence(app.buttons["TabToolbar.stopReloadButton"])
+        XCTAssertEqual(app.buttons["TabToolbar.stopReloadButton"].label, "Search")
+        XCTAssertTrue(app.buttons["TabToolbar.stopReloadButton"].exists)
```

Comment:
This may be better to be checked before checking its label. First check that the button exisits and then check the label, or just leaving the line above would be enough

Reply:
I did this extra step looking for the Search label as I was trying to follow the manual testcase outlined by SV as closely as possible.  Since the label changes, I'm thinking perhaps I should verify it as well??

Reply:
Oh then that's perfect, I was going to suggest that, create the test according to the manual test defined in Test Rail. So yeah, then this double check is fine.